### PR TITLE
Updated README.md file

### PR DESCRIPTION
**Pull request by rkishan516**
README.md file is now updated and it has all the information regarding installing the extension in chrome browser.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
 # Tabs-Share
 
-It is utility chrome extension that helps to save and share a list of tabs opened in the Chrome window at a time. It also provides option to open the saved windows.
\ No newline at end of file
+It is utility chrome extension that helps to save and share a list of tabs opened in the Chrome window at a time. It also provides option to open the saved windows.
+
+### How to Install Tab-Share Extension
+
+	- Download or Clone the extension
```
Comment (akshatnitd) [inline README.md]: Make it `clone`

File: README.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+	- Extract the downloaded File
```
Comment (akshatnitd) [inline README.md]: Make it, `Incase, you have downloaded the code, extract the zip file`

File: README.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+	- Open Chrome Browser
+	- Click on Menu Or Three dots (On Right Corner)
```
Comment (akshatnitd) [inline README.md]: Make it `or`

Comment (akshatnitd) [reply README.md]: Make it `top right corner of the window`

**Comment by rkishan516** [comment]
Completed all the changes

**Review by akshatnitd — COMMENTED** [review]
Please squash the commits and it is ready to merge.

**Review by akshatnitd — APPROVED** [review]

**Pull request merged**